api/location/schema.py

```diff
@@ -68,15 +68,8 @@ def mutate(self, info, location_id, **kwargs):
             validate_timezone_field(**kwargs)
         if "country" in kwargs:
             validate_country_field(**kwargs)
-        if "image_url" in kwargs:
-            validate_url(**kwargs)
-        if "abbreviation" in kwargs:  # noqa
+        if "name" in kwargs or "abbreviation" in kwargs or "imageUrl" in kwargs:  # noqa
```

Comment:
@bevkololi what does this if statement do? What is it checking for?

Reply:
I did not have the latest changes from develop. The above lines were removed. The PR is now updated.